### ISSUE TITLE
Padding?

### ISSUE THREAD

**Issue by mark-in** [tedu/reop#29]
As I can see there is no any padding scheme for message encryption.
There maybe should be one?

**Comment by jpouellet** [comment]
+1 on it being a good idea

**Comment by tedu** [comment]
Yes. You certainly don't want someone to read your messages and see the difference between "yes" and "no" based on length. As a workaround, I recommend "yeah" and "nope" for all communication. :)

This was the last feature cut. I think it requires a message format change, which I've been pondering for a bit as well.
